rust/noosphere-cli/src/native/helpers/cli.rs

```diff
@@ -150,7 +150,8 @@ impl CliSimulator {
         Ok(())
     }
 
-    fn parse_orb_command(&self, command: &[&str]) -> Result<Cli> {
+    /// Parse an orb command slice into a [Cli].
+    pub fn parse_orb_command(command: &[&str]) -> Result<Cli> {
```

Comment:
Any particular reason why this was made public?

Reply:
As a part of tests for this, wanted to parse an orb command and get the storage to assert that some config was set, but had issues with the serialization barrier and also the config only being on the long-running-command `serve` -- I'll roll this back